internalsdk/replica.go

```diff
@@ -126,7 +125,7 @@ func (s *ReplicaServer) newHandler() (*replicaServer.HttpHandler, error) {
 			return s.Flashlight.FeatureOptions(config.FeatureReplica, options)
 		},
 		s.Session.GetCountryCode,
-		geolookup.Refresh)
+		func() {})
```

Comment:
Is making this a no-op okay @anacrolix?

Reply:
@anacrolix geolookup no longer supports forced refresh. The geolocation is now grouped in with the proxy config request.